src/modules/m_joinflood.cpp

```diff
@@ -159,6 +161,8 @@ class ModuleJoinFlood
 		duration = tag->getDuration("duration", 60, 10, 600);
 		bootwait = tag->getDuration("bootwait", 30);
 		splitwait = tag->getDuration("splitwait", 30);
+		denymessage = tag->getString("denymessage", "This channel is temporarily unavailable (+j is set). Please try again later.");
```

Comment:
We should enforce a non-blank message here or the server could, if configured to, send a numeric with no message.
```suggestion
		denymessage = tag->getString("denymessage", "This channel is temporarily unavailable (+j is set). Please try again later.", 1);
```